app/models/department.rb

```diff
@@ -9,7 +9,8 @@
 #
 
 class Department < ApplicationRecord
-  has_and_belongs_to_many :theses
+  has_many :department_theses
```

Comment:
(same singular/plural question)

Reply:
I oddly ran into problems unless I did it this way. I'm confident it was solvable with different variants, but I landed on not really caring enough to figure out why I couldn't use all plural or all singular without jumping through hoops. Sorry I lazied it :/

Reply:
Coolio. Can you leave a comment so we're not confused in future? Then go ahead with the rebase/squash/merge thing.